management/project_charter.md

```diff
@@ -14,22 +14,43 @@
   - [Stakeholders](#stakeholders)
   - [Milestones](#milestones)
   - [Deliverables](#deliverables)
-  - [Budget](#budget) ~~
 
 ## Overview
+This SportShield project aims to enhance the anti-theft device capabilities given by Coris Innovation. It focuses on extending the battery longevity, integrating NFC, optimizing the alarm responsiveness and upgrading security measures. This project aims to raise the device's efficiency, with a rigorous optimization of the software, an exhaustive testing procedure and comprehensive and understandable documentation.
 ## Scope
+
 ## Team Members
 |       |      |      |
 |-------|------|------|
 | Photo | Name | Role |
-|  <img src="https://ca.slack-edge.com/T06NSPENGBX-U06P8AXTKG9-gc3ee79f9544-64">     | Manech LAGUENS | Project Manager |
-|  <img src="https://ca.slack-edge.com/T06NSPENGBX-U06P8F501MF-g01c4dd4777d-64">     | Malo ARCHIMBAUD | Program Manager |
-|   <img src="https://ca.slack-edge.com/T06BWRNMW3X-U06CNHASLBS-gd0f64565afd-64">    | Mattéo LEFFIN | Technical Leader | 
-|   <img src="https://ca.slack-edge.com/T06NSPENGBX-U06NVR88XD0-gc8a06da2c80-64">    | Léo CHARTIER | Software Engineer |
-|   <img src="https://ca.slack-edge.com/T06NSPENGBX-U06NY9A0K0U-ge6ea4153c87-64">    | Laurent BOUQUIN | Technical Writer |
-|   <img src="https://ca.slack-edge.com/T06NSPENGBX-U06P8F4HBL1-g9ecd879a69a-64">    | Mathis KAKAL | Quality Assurance |
+|  <img src="https://ca.slack-edge.com/T06NSPENGBX-U06P8AXTKG9-gc3ee79f9544-64">     | [Manech LAGUENS](https://www.linkedin.com/in/manech-laguens-020127293/) | Project Manager |
+|  <img src="https://ca.slack-edge.com/T06NSPENGBX-U06P8F501MF-g01c4dd4777d-64">     | [Malo ARCHIMBAUD](https://www.linkedin.com/in/malo-archimbaud-58aa12232/) | Program Manager |
+|   <img src="https://ca.slack-edge.com/T06NSPENGBX-U06PJJZ10Q0-gea5dac389e4-64">    | [Mattéo LEFIN](https://www.linkedin.com/in/matt%C3%A9o-lefin-380272293/) | Technical Leader | 
```

Comment:
It would be better if you used Gravatar images instead of pulling the headshots from Slack.